### DISCUSSION TITLE
How to import {Model}CreateInput types?

### DISCUSSION THREAD

**Discussion by sebacruz** [prisma/prisma#15707] [Q&A] [answered]
Hello!

How I import the `CreateInput` types? let's say the `UserCreateInput` type?

I've setup my models and generate the client and I can import models from `@prisma/client` but the `Prisma` namespace doesn't contain the `UserCreateInput` type.

I'm implementing prisma on a nestjs app using `prisma@4.4.0` and `@prisma/client@4.4.0`.

Same that #5241.

**Comment by austincrim** [comment, marked as answer]
Hey @sebacruz 👋,

Those types **should** be included under the `Prisma` type namespace. You can use them like this:
```ts
import { type Prisma } from '@prisma/client'

const input: Prisma.UserCreateInput
```

Are you seeing different behavior?

**Reply by sebacruz** [reply]
Oh, I was trying to extend `Prisma.UserCreateInput`, that was my error.

Thanks!

**Reply by sovetski** [reply]
it did not work for me, even with Prisma type namespace...

PS: I had to use the import from generated prisma instead of `@prisma/client` and it works even without type namespace

`import { PrismaClient, Prisma } from "../generated/prisma";`

**Reply by GustinRhezaRNA** [reply]
I use @prisma/client@6.9.0 and I think the approach changed using 'generated/prisma' as you said in this version
